@xen-orchestra/web-core/lib/components/ui/checkbox/UiCheckbox.vue

```diff
@@ -0,0 +1,396 @@
+<!-- v3 -->
+<template>
+  <!-- @TODO: Replace it by the VtsLabel component when update -->
+  <label class="form-checkbox" v-bind="wrapperAttrs">
+    <input
+      v-model="value"
+      :class="[classNames, { indeterminate: isIndeterminate }]"
+      :disabled="isDisabled"
+      type="checkbox"
+      class="ui-checkbox"
+      v-bind="attrs"
+    />
+    <span class="fake-checkbox">
+      <VtsIcon :fixed-width="false" :icon class="icon" accent="brand" />
```

Comment:
I think the `fixed-width` is not necessary here:
```suggestion
      <VtsIcon :icon class="icon" accent="brand" />
```